syft/frameworks/torch/tensors/interpreters/replicative_shared.py

```diff
@@ -0,0 +1,154 @@
+import torch
+from syft.generic.abstract.tensor import AbstractTensor
+import random
+import syft
+from operator import add, sub
+
+
+class ReplicatedSharingTensor(AbstractTensor):
+    def __init__(
+        self, shares_map=None, owner=None, id=None, tags=None, description=None,
+    ):
+        super().__init__(id=id, owner=owner, tags=tags, description=description)
+        self.child = shares_map
```

Comment:
MAYBE: The ```shares_map``` should be only ```shares```. We can get the ```player``` for a share (or the owner) using the ```location``` attribute for the share

Reply:
This is how it is done in the ```AdditiveSharingTensor```